local_planner/cfg/local_planner_node.cfg

```diff
@@ -12,7 +12,6 @@ gen.add("goal_cost_param_", double_t, 0, "Cost function weight for goal oriented
 gen.add("heading_cost_param_", double_t, 0, "Cost function weight for constant heading", 0.5, 0.5, 20.0)
 gen.add("smooth_cost_param_", double_t, 0, "Cost function wight for path smoothness", 1.5, 0.5, 20.0)
 gen.add("velocity_around_obstacles_", double_t, 0, "Maximally allowed drone velocity in the proximity of obstacles [m/s]", 2.0, 0.0, 5.0)
-gen.add("velocity_far_from_obstacles_", double_t, 0, "Maximally allowed drone velocity when no obstacles are in sight [m/s]", 3.0, 0.0, 10.0)
 gen.add("keep_distance_", double_t, 0, "Distance at which the UAV stops in front of an obstacle", 2.0, 1.0, 8.0)
```

Comment:
My first thought here was that it could be replaced with `MPC_COL_PREV_D`. But I think this is related to the `stop_in_front` functionality? Another legacy piece of code I've been hoping to get rid of  (would then trim down the messy `determineStrategy()`, and also remove the `stop_in_front_` in line 31)